### ISSUE TITLE
[Feature Request] Mi Band 6 support

### ISSUE THREAD

**Issue by drwakey** [piggz/harbour-amazfish#262]
I am not sure if the Mi Band 6 should be able to connect with the Mi Band 4 profile, but at least on my XA2 with Sailfish OS 4.5 i could not get it to pair / be found by Amazfish.
I do have the proper auth key, pairing the band to Gadgetbridge with my old Android Tablet was successful.

To be at least able to use some of it's features with my phone, would be a blast :-)

**Comment by kan-ibal** [comment]
Yeah, support for Mi Band 7 would good as well. I suppose that it is the same issue as for 6. Mi Band 7 is the last Mi Band compatible with amazfit/zepp.

**Comment by AntonIXO** [comment]
If they are all use almost identical protocol, could we add connections support and some basic features like step syncing would work? There was a pr that added just a pic and mi band 5 worked)
